examples/guestbook-demo/db.yaml

```diff
@@ -1,18 +1,20 @@
 name: mongodb
 
-containers:
-  - image:  centos/mongodb-32-centos7:3.2
-    envFrom:
-      - secretRef:
-          name: mongodb-admin
-      - secretRef:
-          name: mongodb-user
-      - configMapRef:
-          name: mongodb-user
+deployments:
+- containers:
```

Comment:
yeah.. looking at previous examples, I believe it should not be an array, just `containers`...

Reply:
this should be an array. The one where it is not is wrong. (my mistake)

It is quite possible there will be more stuff that I might miss, it was a lot of examples to update.